docs/ldml/tr35.md

```diff
@@ -242,6 +242,14 @@ External specifications may also reference particular components of Unicode loca
 
 > _Field X can contain any Unicode region subtag values as given in Unicode Technical Standard #35: Unicode Locale Data Markup Language (LDML), excluding grouping codes._
 
+### <a name="String_Identifiers" href="#String_Identifiers">String Identifiers</a>
+
+Unicode LDML specifies many enumerations of string identifiers, such as days of the week ("`mon"`, `"tue"`, …), units of measurement (`"meter"`, `"inch"`, …), and person name fields (`"title"`, `"given"`, …). For any such string enumeration, the following identifiers are reserved for use by implementations and will not be added as explicit entries into the enumeration by LDML:
```

Comment:
I think "specifies many enumerations" is too vague to be normative. 

One way to more completely specify it would be to restrict it to attribute values. That is, however, too broad, because attributes can be representing other information. Or maybe focus on the terms that can be in bcp47 subtags?

One way to make progress in v44 would be to explicitly list _those enumerations_ that cannot contain these terms, and then in v45 work on a more general solution.